api/openapi-spec/swagger.json

```diff
@@ -19323,7 +19323,7 @@
   },
   "info": {
     "title": "Kubernetes",
-    "version": "v1.15.0"
+    "version": "v0.0.0"
```

Comment:
Need to revert this. BTW, why the version is updated to "0.0"? the Arktos is at 0.2.

I will try to get the PR to fix the update on swagger.json merged next week.

Reply:
this is auto-generated. what's the best way to revert this other than manually fixing it?

Reply:
once https://github.com/futurewei-cloud/arktos/pull/246 is in, we will be free of this nuisance.